ui-lib/src/androidTest/java/co/electriccoin/zcash/ui/common/model/VersionInfoTest.kt

```diff
@@ -0,0 +1,25 @@
+package co.electriccoin.zcash.ui.common.model
+
+import androidx.test.filters.SmallTest
+import co.electriccoin.zcash.ui.test.getAppContext
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotEquals
+import kotlin.test.assertTrue
+
+class VersionInfoTest {
+
+    @Test
+    @SmallTest
+    fun sanity_check_version_info_in_testing() {
+        val versionInfo = VersionInfo.new(getAppContext())
+
+        // We expect few of its parameter to be not set during testing
```

Comment:
This sentence is grammatically wrong, I don't understand it. Probably something like

`We expect a few parameters to be set during the testing` or `We expect only a few parameters to not be set during the testing`

Reply:
Comment updated.